VAMobile/src/testUtils.tsx

```diff
@@ -233,8 +233,12 @@ ctxFn.skip = (name: string, fn: () => void) => {
 
 export const context = ctxFn
 
+export type QueriesData = Array<{
+  queryKey: QueryKey
+  data: any
+}>
 //@ts-ignore
-function render(ui, { preloadedState, navigationProvided = false, ...renderOptions } = {}) {
+function render(ui, { preloadedState, navigationProvided = false , ...renderOptions } = {}, queriesData?: QueriesData) {
```

Comment:
Yeah I agree, originally I had it within the object in the 2nd param, but something threw me off with the types, so I moved it to the end of the function. It definitely makes it easier to read and cleaner having it there though. I just moved it back 👍. And initially I had it as a single object named `queryData` instead of an array of objects, but decided to make an array, so it can handle setting up data for multiple queries just in case. Now that I think about it, we could also update it to be a single object where the `key` is the query key and the `value` is the data